Source/Charts/Data/Implementations/Standard/ChartDataSet.swift

```diff
@@ -55,12 +56,18 @@ open class ChartDataSet: ChartBaseDataSet
     /// - Note: Calls `notifyDataSetChanged()` after setting a new value.
     /// - Returns: The array of y-values that this DataSet represents.
     /// the entries that this dataset represents / holds together
-    @available(*, unavailable, renamed: "entries")
-    @objc
-    open var values: [ChartDataEntry] { return entries }
-
     @objc
     open private(set) var entries: [ChartDataEntry]
+    {
+        didSet
+        {
+            if isIndirectValuesCall {
```

Comment:
@jjatie one of the main blocker now is this `isIndirectValuesCall`. I have removed the old `values` var to make it simple 

Could you take a look if it can be removed now?

Reply:
@jjatie this is kind of higher priority, not sure if can remove it safely now?

Reply:
I made the necessary changes to remove it.